using_images/other_images/jenkins.adoc

```diff
@@ -169,51 +162,109 @@ To create a new Jenkins application using standard Jenkins authentication:
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
-    openshift/jenkins-1-centos7
+    openshift/jenkins-2-centos7
 ----
 
 [[jenkins-environment-variables]]
 === Environment Variables
 
 The Jenkins server can be configured with the following environment variables:
 
-.Jenkins Environment Variables
-[cols="4a,6a",options="header"]
-|===
+* `JENKINS_PASSWORD` (default: `password`)
++
+The password for the `admin` user when using standard Jenkins authentication.
+Not applicable when using {product-title} OAuth authentication.
 
-|Variable name |Description
+* `OPENSHIFT_ENABLE_OAUTH` (default: `false`)
++
+Determines whether the OpenShift Login plug-in manages authentication when
+logging into Jenkins. To enable, set to `true`.
 
-|`*JENKINS_PASSWORD*`
-|The password for the `*admin*` user when using standard Jenkins authentication.
-Not applicable when using {product-title} OAuth authentication.
+* `CONTAINER_HEAP_PERCENT` (default: `0.5`, i.e. 50%)
++
+Specifies the percentage of the container memory limit allocated to the Jenkins
+JVM maximum heap size (-Xmx).
+
+* `JENKINS_MAX_HEAP_UPPER_BOUND_MB`
++
+If set, specifies a static upper bound in MiB for the Jenkins JVM maximum heap
+size (-Xmx).  The effective setting used is the minimum of
+`CONTAINER_HEAP_PERCENT` and `JENKINS_MAX_HEAP_UPPER_BOUND_MB`. Example setting:
+`512`.
+
+* `JENKINS_JAVA_OPTIONS`
++
+Specifies additional options for the Jenkins JVM which may be overridden.
 
-|`*OPENSHIFT_ENABLE_OAUTH*`
-|Determines whether the OpenShift Login plug-in manages authentication when
-logging into Jenkins. Enabled when set to any non-empty value other than
-"false".
+* `JENKINS_JAVA_OVERRIDES`
++
+Specifies additional options for the Jenkins JVM which may not be overridden.
```

Comment:
I think what really needs to be said here is that these JVM options will appear last in the JVM arguments, meaning they will/can override arguments provided earlier (such as via the automatically generated arguments that appear in JENKINS_JAVA_OPTIONS, or the arguments set by the user in JENKINS_JAVA_OPTIONS if they chose to set it explicitly).

Also worth distinguishing that unlike JENKINS_JAVA_OPTIONS, there is no "default/automatic" value for JENKINS_JAVA_OVERRIDES so if they choose to set this value, they are not erasing any arguments the image would have normally supplied.  (They may, however, be overriding values the image is trying to supply).